fusor-ember-cli/app/controllers/openshift/openshift-configuration.js

```diff
@@ -1,12 +1,17 @@
 import Ember from 'ember';
-import NeedsDeploymentMixin from "../../mixins/needs-deployment-mixin";
+import NeedsDeploymentMixin from '../../mixins/needs-deployment-mixin';
+import ValidatesMounts from '../../mixins/validates-mounts';
 import {
   RequiredPasswordValidator,
   EqualityValidator,
   validateZipper
 } from '../../utils/validators';
 
-export default Ember.Controller.extend(NeedsDeploymentMixin, {
+export default Ember.Controller.extend(NeedsDeploymentMixin, ValidatesMounts, {
+
+  errorMsg: null,
```

Comment:
Do these need to be in the route setupController since we want to initialize them everytime the page is loaded?